Packs/EWS/Integrations/EWSO365/EWSO365.py

```diff
@@ -0,0 +1,1765 @@
+        return {
+            MAILBOX: element.find(f"{{{TNS}}}PrimarySmtpAddress").text
+            if element.find(f"{{{TNS}}}PrimarySmtpAddress") is not None
+            else None,
+            MAILBOX_ID: element.find(f"{{{TNS}}}ReferenceId").text
+            if element.find(f"{{{TNS}}}ReferenceId") is not None
+            else None,
+            "displayName": element.find(f"{{{TNS}}}DisplayName").text
+            if element.find(f"{{{TNS}}}DisplayName") is not None
+            else None,
+            "isExternal": element.find(f"{{{TNS}}}IsExternalMailbox").text
+            if element.find(f"{{{TNS}}}IsExternalMailbox") is not None
+            else None,
+            "externalEmailAddress": element.find(f"{{{TNS}}}ExternalEmailAddress").text
+            if element.find(f"{{{TNS}}}ExternalEmailAddress") is not None
+            else None,
+        }
+
+    def call(self):
+        elements = self._get_elements(payload=self.get_payload())
+        return [
+            self.parse_element(x)
+            for x in elements
+            if x.find(f"{{{TNS}}}ReferenceId").text
+        ]
+
+    def get_payload(self):
+        element = create_element(f"m:{self.SERVICE_NAME}")
+        return element
+
+
+class ExpandGroup(EWSService):
+    SERVICE_NAME = "ExpandDL"
+    element_container_name = f"{{{MNS}}}DLExpansion"
+
+    @staticmethod
+    def parse_element(element):
+        return {
+            MAILBOX: element.find(f"{{{TNS}}}EmailAddress").text
+            if element.find(f"{{{TNS}}}EmailAddress") is not None
+            else None,
+            "displayName": element.find(f"{{{TNS}}}Name").text
+            if element.find(f"{{{TNS}}}Name") is not None
+            else None,
+            "mailboxType": element.find(f"{{{TNS}}}MailboxType").text
+            if element.find(f"{{{TNS}}}MailboxType") is not None
+            else None,
+        }
+
+    def call(self, email_address, recursive_expansion=False):
+        try:
+            if recursive_expansion == "True":
+                group_members = {}  # type: dict
+                self.expand_group_recursive(email_address, group_members)
+                return list(group_members.values())
+            else:
+                return self.expand_group(email_address)
+        except ErrorNameResolutionNoResults:
+            demisto.results("No results were found.")
+            sys.exit()
+
+    def get_payload(self, email_address):
+        element = create_element(f"m:{self.SERVICE_NAME}")
+        mailbox_element = create_element("m:Mailbox")
+        add_xml_child(mailbox_element, "t:EmailAddress", email_address)
+        element.append(mailbox_element)
+        return element
+
+    def expand_group(self, email_address):
+        elements = self._get_elements(payload=self.get_payload(email_address))
+        return [self.parse_element(x) for x in elements]
+
+    def expand_group_recursive(self, email_address, non_dl_emails, dl_emails=set()):
```

Comment:
better to avoid settings args default as mutable, use `None` instead and assign it in the function as `set()` if `None`